src/main/java/carracing/domain/car/Car.java

```diff
@@ -0,0 +1,61 @@
+package carracing.domain.car;
+
+import carracing.domain.move.MoveStrategy;
+
+import java.util.Objects;
+
+public class Car {
+    private final Name name;
+    private final Position position;
+
+    private Car(Name name, Position position) {
+        this.position = position;
+        this.name = name;
+    }
+
+    public static Car of(Name name, Position position) {
+        return new Car(name, position);
+    }
+
+    public static Car from(Name name) {
+        return new Car(name, Position.create());
+    }
+
+    public Car move(MoveStrategy strategy) {
+        if (strategy.isMovable()) {
+            Position increasedPosition = Position.from(position.increase());
```

Comment:
```suggestion
            Position increasedPosition = position.increase();
```
이렇게도 사용해 볼 수 있을 것 같아요

Reply:
더 좋은 방법이 있었네요!👍👍